Assets/HoloToolkit/Input/Tests/Scripts/SimpleGridGenerator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class SimpleGridGenerator : MonoBehaviour
```

Comment:
Could you share what wasn't fun about keeping cross-folder dependencies in separate folders?

This is what I thought:
HoloToolkit/UnitTests/Input - would depend only on HoloToolkit/Input
HoloToolkit/IntegrationTests/Input - could have cross-folder dependencies
HoloToolkit/Examples/Input - could have cross-folder dependencies

I think additional examples and tests could probably outweigh a potential inconvenience of having to delete a few more folders once in a project lifecycle.

Another way to do this is potentially create a separate repository (say HoloToolkit-Unity-Examples) which could point to HoloToolkit-Unity as a submodule.